android/src/main/java/com/mapbox/mapboxgl/MapBoxUtils.java

```diff
@@ -0,0 +1,41 @@
+package com.mapbox.mapboxgl;
+
+import android.content.Context;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.mapbox.mapboxsdk.Mapbox;
+
+abstract class MapBoxUtils {
+    private static final String TAG = "MapboxMapController";
+
+    static Mapbox getMapbox(Context context) {
```

Comment:
Can we remove this constructor, as it isn't used and probably shouldn't be used without an access token?